src/stores/configurationStore.ts

```diff
@@ -41,15 +41,15 @@ export const useConfigurationStore = defineStore('configuration', {
     },
 
     getCreateUrl (state: ConfigurationStateIF): string {
-      return state.configuration.VUE_APP_BUSINESS_CREATE_URL
+      return state.configuration?.VUE_APP_BUSINESS_CREATE_URL
```

Comment:
This omission (also above and below) caused a lot of warnings in the unit tests because the configuration is not always (or ever) populated there.